.github/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 1. AIRAC (2407) - Changed color of Fast Aviation taxiway on Bournemouth (EGHH) SMR - thanks to @kristiankunc (Kristián Kunc)
 2. AIRAC (2407) - Updated Jersey (JSY) VOR coordinates - thanks to @chssn
 3. AIRAC (2407) - Removed Lashenden NDB (LSH) - thanks to @Bucklerz (Reece Buckley)
+X. AIRAC (2408) - Add Leeds Bradford (EGNM) Tug Release Points - thanks to @RedstonePilot (Ben Walker)
```

Comment:
Note to merger: Remember this is a 2408 issue so changelog entry will have to go in new section after 2407 released.